west.yml

```diff
@@ -98,6 +101,10 @@ manifest:
       repo-path: sdk-nrfxlib
       path: nrfxlib
       revision: 761bcbfc6223e232687af93ffe53a653bd0e02ea
+    - name: hal_nordic
+      path: modules/hal/nordic
+      remote: wbober
+      revision: add_ble_to_zigbee_light_switch
```

Comment:
Could you update to point to another PR, if this is not available on master ?

Reply:
fixed